src/main/java/org/jabref/gui/preferences/websearch/WebSearchTabViewModel.java

```diff
@@ -181,6 +181,13 @@ public BooleanProperty getApikeyPersistProperty() {
     }
 
     public void checkCustomApiKey() {
+        if (selectedApiKeyProperty.get() == null) {
+            dialogService.showErrorDialogAndWait(
+                    Localization.lang("No API was selected"),
+                    Localization.lang("Please select an API first to be tested."));
+            return;
+        }
```

Comment:
Please disable the button if no API key (row in the table!!) is selected instead of outputting a dialog. Thus, we also save new translatations.